lib/smart_answer/calculators/pip_dates.rb

```diff
@@ -12,7 +12,7 @@ def in_group_65?
     end
 
     def in_middle_group?
-      self.dob > GROUP_65_CUTOFF && self.dob < MIDDLE_GROUP_CUTOFF
+      self.dob > GROUP_65_CUTOFF && self.dob <= 16.years.ago
```

Comment:
@ikennaokpala As you are no longer using variable MIDDLE_GROUP_CUTOFF here, and it's not being used anywhere else, I think it should be removed completely.